common/djangoapps/track/tests/test_contexts.py

```diff
@@ -72,3 +87,20 @@ def test_course_id_later_in_url(self, course_id, postfix):
 
     def test_no_url(self):
         self.assert_empty_context_for_url(None)
+
+    def enterprise_context_from_course_id_user_id(self):
+        if ENTERPRISE_ENABLED and enterprise_enabled():
+            user = UserFactory()
+            course_enrollment = CourseEnrollmentFactory(mode=CourseMode.VERIFIED, user=user)
+            course = course_enrollment.course
+            if is_enterprise_learner(user):
```

Comment:
@rehanedly why we need this if statement here?

Reply:
fixed